Issue1916 boiler plant enabler

milicag opened the pull request:
Closes #1916

milicag commented:
@karthikeyad-pnnl I've introduced a few smaller changes, please see what those are. Once I hear from you that those changes are clear to you I'll merge and update 1925 branch for you to merge into the next dev branch.

karthikeyad-pnnl commented:
@milicag Thanks! I saw the changes you had made as per your comments about the checklist in your email. How did you find the blankspace at the end of the lines that you changed in the latest commit?
Also, I noticed that the order of variables in the reference results changed. Is that something that happens because of the compiler?

All the changes make sense. You can merge the branch. Appreciate your help and guidance with this pull request!

Thanks,
Karthik

milicag commented:
@karthikeyad-pnnl I think my dymola did the blank spaces in the latest commit, so that I don't know. I deleted the results file and regenerated it when I've changed the results file name, so likely the compiler. Looking forward to the next one.
